src/services/file/fixtures.ts

```diff
@@ -0,0 +1,37 @@
+import { FileServiceUrlCaching } from './caching';
```

Comment:
I think you can also mock with a folder like `_mock_` https://jestjs.io/docs/manual-mocks so it's automatic maybe that's what you want?